cpp/config/Make.rules

```diff
@@ -14,9 +14,8 @@
 slice2cpp_targetext     = cpp
 slice2cpp_path          = $(ice_bindir)/slice2cpp
 
-# Use the C++14 standard
-cppflags                += -std=c++14
-
+# Use the C++20 standard
+cppflags                += -std=c++20
```

Comment:
We should use the same as for Ice (obviously, since the goal is to merge DataStorm into the Ice build). This is presumably temporary.

Reply:
Yes, this is temporary, once we merge into Ice we would use the Make.rules from Ice repository.